ocean_provider/validation/RBAC.py

```diff
@@ -77,6 +78,17 @@ def get_additional_dids(self):
             for additional_input in additional_inputs
         ]
 
+    def get_data(self):
+        if "data" not in self.request.keys():
+            raise Exception("Data to encrypt is empty.")
+        # import pdb;pdb.set_trace()
+        if not isinstance(self.request["data"], list) and not isinstance(
```

Comment:
while request /encrypt hit, both asset files encryption and ddo encryption will send to RBAC and let RBAC decide what to do next?